src/app/features/cases/components/pages/edit/hooks/useEditPage.ts

```diff
@@ -27,7 +24,7 @@ export const useEditPage = (uuid?: API.Case["uuid"]) => {
     }
   })
 
-  const handleDelete = useCrudDelete(caseDetails!, {
+  const handleDelete = useCrudDelete(initialValues!, {
```

Comment:
Can we maybe add a guard to `useCrudDelete` to handle undefined initialValues? Instead of the not nullish casting.